src/components/Router/HistoryRouter.tsx

```diff
@@ -45,7 +50,20 @@ export const HistoryRouterWrapper = ({
     )
   }
 
-  return <HistoryRouter {...props} {...useStepsProps} steps={steps} />
+  return (
+    <SdkConfigurationServiceContext.Consumer>
```

Comment:
Any reason you're using a consumer instead of the hook inside the `HistoryRouter` itself?

Reply:
Right, I forgot about that 🤦

Reply:
Reverted as we don't need it anymore with `usePassiveSignals()` custom hook